<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlanWindow</class>
 <widget class="QMainWindow" name="PlanWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>841</width>
    <height>680</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PlanWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="addButton">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>570</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить строку</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteButton">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>570</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Удалить строку</string>
    </property>
   </widget>
   <widget class="QComboBox" name="VkUvcComboBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="VUSComboBox">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>40</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="semesterComboBox">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>40</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="disciplineComboBox">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>40</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="platoonComboBox">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>40</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>801</width>
      <height>481</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="reportButton">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>600</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Отчет</string>
    </property>
   </widget>
   <widget class="QPushButton" name="leftButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>570</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rightButton_2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>570</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="pageCountLabel">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>570</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="currentPageEdit">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>571</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>841</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>addButton</sender>
   <signal>clicked()</signal>
   <receiver>PlanWindow</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>650</x>
     <y>618</y>
    </hint>
    <hint type="destinationlabel">
     <x>470</x>
     <y>638</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
